lang/en-EN/game.xml

```diff
@@ -1905,7 +1905,9 @@
 			<story_3><![CDATA[<b>100 Patients healed:</b>⮀ You've healed some of these townsfolk so many times, you don't actually need to speak to them or examine them to find out what's wrong. You go through the same steps anyways, because good bedside manners are an integral part of earning respect as a healer.]]></story_3>
 			<story_4><![CDATA[<b>1000 Patients healed:</b>⮀ You never formally trained as a healer, but you've healed so many people that you're pretty much an expert now. Nothing can stop you, nothing can stump you, nothing can even slow you down! ...Except when they bring in a patient you haven't seen before. When that happens, it usually takes you a while to figure out what their problem is.]]></story_4>
 			<story_5><![CDATA[<b>Failed Action:</b>⮀ You're turned away at the door, the small healers home stating that they can't trust you to not poison the people inside or make the situation worse.]]></story_5>
-			<story_6><![CDATA[<b>50 Restoration:</b>⮀ You'd say it's like magic, but you were already using magic so instead it's like double-magic. The medics can't tell what you're doing, it's completely outside their paradigm: you simply touch the patient's forehead and, with a surge of energy, they're healed. You don't do it to boast, though. All you want is to help.]]></story_6>
+			<!-- Story 6 was moved to 8. Don't use that number or your stories may not show in the action log properly for old saves. -->
+            <story_7><![CDATA[<b>1 Restoration:</b>⮀ You attempt to use your new Restoration magic to help heal, but your fumbling is more likely to harm rather than heal.  You decide to try again when you've had more practice.]]></story_6>
+            <story_8><![CDATA[<b>50 Restoration:</b>⮀ You'd say it's like magic, but you were already using magic so instead it's like double-magic. The medics can't tell what you're doing, it's completely outside their paradigm: you simply touch the patient's forehead and, with a surge of energy, they're healed. You don't do it to boast, though. All you want is to help.]]></story_7>
```

Comment:
Don't rewrite numbers, that isn't required anymore. Look elsewhere in this file for how to use the `<story>` tag and rewrite all of the stories for this action, putting the numbers out of order as necessary.